### PR TITLE
Python Tutorial Updates from doc read

### PR DESCRIPTION
**Pull request by cwebster-99**
Updating Python tutorial from Python team doc read through

### CODE REVIEW

File: docs/python/python-tutorial.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Along with the Python extension, you need to install a Python interpreter. Which
 
 Install [Python from python.org](https://www.python.org/downloads/). You can typically use the **Download Python** button that appears first on the page to download the latest version.
 
->**Note**: If you don't have admin access, an additional option for installing Python on Windows is to use the Microsoft Store. The Microsoft Store provides installs of [Python 3.7](https://www.microsoft.com/p/python-37/9nj46sx7x90p), [Python 3.8](https://www.microsoft.com/p/python-38/9mssztt1n39l), [Python 3.9](https://www.microsoft.com/p/python-39/9p7qfqmjrfp7), and [Python 3.10](https://www.microsoft.com/p/python-310/9pjpw5ldxlz5).
+>**Note**: If you don't have admin access, an additional option for installing Python on Windows is to use the Microsoft Store. The Microsoft Store provides installs of [Python 3.7](https://www.microsoft.com/p/python-37/9nj46sx7x90p), [Python 3.8](https://www.microsoft.com/p/python-38/9mssztt1n39l), [Python 3.9](https://www.microsoft.com/p/python-39/9p7qfqmjrfp7), [Python 3.10](https://www.microsoft.com/p/python-310/9pjpw5ldxlz5), and [Python 3.11](https://apps.microsoft.com/store/detail/python-311/9NRWMJP3717K?hl=en-us&gl=us).
```
Comment (brettcannon) [inline docs/python/python-tutorial.md:48]: This is going to be a losing battle. I think we should just link to the Python Software Foundation publisher account which lists all versions of Python available: https://apps.microsoft.com/store/search?hl=en-us&gl=us&publisher=Python%20Software%20Foundation .

**Review by brettcannon — APPROVED** [review]

File: docs/python/python-tutorial.md [hunk 2]
```diff
@@ -4,23 +4,23 @@ Area: python
...
-This tutorial introduces you to VS Code as a Python environment, primarily how to edit, run, and debug code through the following tasks:
+This tutorial introduces you to VS Code as a Python environment - primarily how to edit, run, and debug code through the following tasks:
 
 - Write, run, and debug a Python "Hello World" Application
 - Learn how to install packages by creating Python virtual environments
 - Write a simple Python script to plot figures within VS Code
 
 This tutorial is not intended to teach you Python itself. Once you are familiar with the basics of VS Code, you can then follow any of the [programming tutorials on python.org](https://wiki.python.org/moin/BeginnersGuide/Programmers) within the context of VS Code for an introduction to the language.
 
-If you have any problems, you can search for answers or ask a question on the [Python extension Discussions Q&A](https://github.com/microsoft/vscode-python/discussions/categories/q-a).
```
Comment (gregvanl) [inline docs/python/python-tutorial.md:23]: This was changed recently per Luciana's recommendation to point to the Python discussions.
https://github.com/microsoft/vscode-docs/commit/5f4987e50431e5faa1dbaa5b45fc9c7de9c1244f
Are you sure you want to go back to vscode-docs issues?

**Comment by gregvanl** [comment]
This PR reverts several other recent PRs so marking at `don't merge` until this is sorted out.

File: docs/python/python-tutorial.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ The built-in Python 3 installation on Linux works well, but to install other Pyt
...
+- **Windows Subsystem for Linux**: If you are working on Windows and want a Linux environment for working with Python, the [Windows Subsystem for Linux](https://docs.microsoft.com/windows/wsl/about) (WSL) is an option for you. If you choose this option, you'll also want to install the [Remote - WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl). For more information about using WSL with VS Code, see [VS Code Remote Development](/docs/remote/remote-overview.md) or try the [Working in WSL tutorial](/docs/remote/wsl-tutorial.md), which will walk you through setting up WSL, installing Python, and creating a Hello World application running in WSL.
```
Comment (gregvanl) [inline docs/python/python-tutorial.md:66]: docs.microsoft.com was renamed to learn.microsoft.com back in Sept 2022.
I don't think we want to revert those changes (more below)
Also the WSL extension was renamed from "Remote-WSL" also in Sept so we definitely don't want to revert that change

**Pull request closed without merging**